examples/shared/BUILD

```diff
@@ -6,19 +6,12 @@ load("//sass:sass.bzl", "sass_library")
 sass_library(
     name = "colors",
     srcs = ["_colors.scss"],
+    visibility = ["//visibility:public"],
 )
 
 # make a :fonts target that any sass_binary rules can depend on.
 sass_library(
     name = "fonts",
     srcs = ["_fonts.scss"],
-)
-
-# make a :shared target that includes both :colors and :fonts.
-sass_library(
```

Comment:
Gazelle doesn't make a `:shared` target so I deleted it by hand. Note that if I hadn't deleted it, gazelle would have left it alone and it would still be there. Gazelle only updates rules that it would have created if the directory were empty.